app/views/content/train-to-be-a-teacher/_questions.html.erb

```diff
@@ -0,0 +1,28 @@
+
+<section class="category__questions">
+    <div class="category__question">
+        <h2>
+            <strong>What subject</strong> do you want to teach?
+        </h2>
+
+        <p>
+            If you want to teach a particular subject there may be extra
```

Comment:
As per the heading, I don't think this is quite right in terms of setting expectations